feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-method.component.ts

```diff
@@ -36,20 +36,23 @@ import { CheckoutStepService } from '../services/checkout-step.service';
 })
 export class CheckoutPaymentMethodComponent implements OnInit, OnDestroy {
   protected subscriptions = new Subscription();
+  protected deliveryAddress: Address | undefined;
+  protected busy$ = new BehaviorSubject<boolean>(false);
 
   iconTypes = ICON_TYPE;
   existingPaymentMethods$: Observable<PaymentDetails[]>;
-  isLoading$: Observable<boolean>;
   cards$: Observable<{ content: Card; paymentMethod: PaymentDetails }[]>;
   selectedMethod$: Observable<PaymentDetails | undefined>;
   isGuestCheckout = false;
   newPaymentFormManuallyOpened = false;
-  paymentSavingInProgress$ = new BehaviorSubject<boolean>(false);
-  shouldRedirect: boolean;
+  shouldRedirect: boolean = false;
 
   backBtnText = this.checkoutStepService.getBackBntText(this.activatedRoute);
 
-  protected deliveryAddress: Address | undefined;
+  isUpdated$: Observable<boolean> = combineLatest([
```

Comment:
I think we chose the wrong name in the delivery-address cmp. It seems better to name it `isUpdating$`. What do you think?